src/modules/iasql_functions/rpcs/iasql_get_sql_since.ts

```diff
@@ -0,0 +1,387 @@
+          cl.tableName,
+          ...updatedEntries.flatMap(([k, _]: [string, any], i) => [k, updatedValues[i]]),
+          ...originalEntries.flatMap(([k, _]: [string, any], i) => [k, oldValues[i]]),
+        );
+        break;
+      default:
+        break;
+    }
+    if (query) queries.push(query);
+  }
+  return queries;
+}
+
+/**
+ * @internal
+ * The changes from iasql_audit_log are stored as JSON.
+ * We need to look if the value is a relation to other table and return the respective sub-query
+ * or return a formatted value for the query.
+ */
+async function findRelationOrReturnFormattedValue(
+  tableName: string,
+  key: string,
+  value: any,
+  modsIndexedByTable: { [key: string]: ModuleInterface },
+  orm: TypeormWrapper,
+): Promise<string> {
+  // We might need to recreate a sub-query because it could be column referencing other table.
+  // For this we need to get Typeorm metadata for the `tableName` and inspect the columns and relations in order to recreate the sub-query if necessary.
+  // We need to recreate the sub-query because related columns might not be the same across databases connected to the same cloud account.
+  const metadata = await getMetadata(tableName, modsIndexedByTable, orm);
+  let columnMetadata: ColumnMetadata | undefined;
+  if (value && metadata) {
+    // If `metadata instanceof EntityMetadata` means that there's an Entity in Typeorm which it's table name is `tableName`
+    if (metadata instanceof EntityMetadata) {
+      columnMetadata = metadata.ownColumns
+        .filter(oc => oc.databaseName === key)
+        .map(oc => oc)
+        ?.pop();
+      if (!!columnMetadata?.relationMetadata) {
+        return await recreateSubQuery(
+          columnMetadata.referencedColumn?.databaseName ?? 'unknown_key',
+          value,
+          columnMetadata.relationMetadata?.inverseEntityMetadata,
+          modsIndexedByTable,
+          orm,
+        );
+      }
+    }
+    // If `metadata instanceof RelationMetadata` means that there's no Entity in Typeorm which it's table name is `tableName`,
+    // but theres a join table linking entities and `tableName` is that join table. In this case we need to check `joinColumns`
+    // which will have the columns from the owner of the relationship and `inverseJoinColumns` will have the columns coming from
+    // the other entities in the relationship.
+    if (metadata instanceof RelationMetadata) {
+      columnMetadata = metadata.joinColumns
+        .filter(jc => jc.databaseName === key)
+        .map(jc => jc)
+        ?.pop();
+      if (!!columnMetadata?.relationMetadata) {
+        return await recreateSubQuery(
+          columnMetadata.referencedColumn?.databaseName ?? 'unknown_key',
+          value,
+          columnMetadata.relationMetadata?.entityMetadata,
+          modsIndexedByTable,
+          orm,
+        );
+      }
+      columnMetadata = metadata.inverseJoinColumns
+        .filter(jc => jc.databaseName === key)
+        .map(jc => jc)
```

Comment:
![and another one](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.giphy.com%2Fmedia%2FGV3aYiEP8qbao%2Fgiphy.gif&f=1&nofb=1&ipt=4cdb0afc33b897b628180d0637e8e1a6041a402940e342526606f1dbcafd687a&ipo=images)